src/App.js

```diff
@@ -1,7 +1,29 @@
 import "./App.css";
+import Mockman from "mockman-js";
```

Comment:
Try to follow the convention of global imports first then local imports

Reply:
didnt knew about this thanks from now will follow that 👍